plugins/templates/templates/default.js

```diff
@@ -15,13 +15,13 @@ CKEDITOR.addTemplates( 'default', {
red list, don\'t use subscripts arr<sub>i</sub>, use arr[i] instead</li><li>subscripts are too small to see sometimes, e.g. a<sub>i</sub> versus a<sub>j</sub></li></ul><p> </p><details><summary class="section-title" aria-level="4" role="heading">Input Format for Custom Testing</summary><div class="collapsable-details"><p>Input from stdin will be processed as follows and passed to the function.</p><p> </p><p>The first line contains an integer <em>n</em>, the size of the array <em>numbers</em>.</p><p>Each of the next <em>n</em> lines contains an integer <em>numbers[i]</em>. (no subscripts, they\'re too small to read)</p><p>With the exception of the temporary values, e.g. array size which isn\'t passed to the candidates, describe only the input. The use of the inputs is described in the Function Description and does not need to be repeated here.</p><p> </p></div></details><details open="open"><summary class="section-title" aria-level="4" role="heading">Sample Case 0</summary><div class="collapsable-details"><p aria-level="5" role="heading" class=”section-title”><strong>Sample Input 0</strong></p><pre> STDIN Function ----- -------- < as an example, a single element array is input> 1 the size of arr n = 1 2 arr = [2] </pre><p aria-level="5" role="heading" class=”section-title”><strong>Sample Output 0</strong></p><pre> output exactly as it is expected </pre><p aria-level="5" role="heading" class=”section-title”><strong>Explanation</strong></p><p><strong>As a last step, always test the challenge with a complete solution.</strong>If you have created custom code stubs, test for proper I/O for all languages at a minimum. It is easy to let the code stub read and pass the parameters, then return a dummy value. This saves a lot of effort in the long run.</p><p> </p><p>Sample 0 should be expanded when the question opens. Open Sample 0, close the other sections and press <source> at the top of this window to automatically add open="" to the details tag for this section. While in the source view, edit the style to open="open". You can manually insert the tag in the source window too.</p><p>Explanatory text and diagrams (svg). Like the Example section, keep the text to a minimum. Note that the Sample Input and Output sections show input and expected results clearly. They generally do not need to be repeated here.</p></div></details><details><summary class="section-title" aria-level="4" role="heading">Sample Case 1</summary><div class="collapsable-details"><p aria-level="5" role="heading" class=”section-title”><strong>Sample Input 1</strong></p><pre> STDIN Function ----- -------- < as an example, a single element array called arr is the input> 1 the size of arr[] n = 1 2 arr = [2] </pre><p aria-level="5" role="heading" class=”section-title”><strong>Sample Output 1</strong></p><pre> output exactly as it is expected </pre><p aria-level="5" role="heading" class=”section-title”><strong>Explanation</strong></p><p>Explanatory text and diagrams</p></div></details></div>'
```

Comment:
Shouldn't we wrap the content in html and body tags ?

Reply:
Done, made the required changes